packages/vats/src/core/boot-psm.js

```diff
@@ -179,6 +181,8 @@ export const buildRootObject = (vatPowers, vatParameters) => {
       buildZoe(powersFor('buildZoe')),
       makeBoard(powersFor('makeBoard')),
       makeBridgeManager(powersFor('makeBridgeManager')),
+      noProvisioner(powersFor('makeProvisioner')),
```

Comment:
This looks like a mistake, by granting too much power to `noProvisioner`.  At the same time, noted below, the `noProvisioner` entry should not go in the manifest that is used by `core/boot.js`.  Please lift out that entry and put it explicitly in `boot-psm.js` so that `powersFor('noProvisioner')` references it.

Reply:
Done